Bugsnag/Payload/BugsnagBreadcrumb+Private.h

```diff
@@ -20,4 +20,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+NSString *BSGBreadcrumbTypeValue(BSGBreadcrumbType type);
+BSGBreadcrumbType BSGBreadcrumbTypeFromString( NSString * _Nullable value);
+
+#ifdef __cplusplus
+}
+#endif
+
```

Comment:
We could use the `FOUNDATION_EXPORT` macro to take care of the `extern "C"` dance thereby tidying up the code a bit:

```suggestion
FOUNDATION_EXPORT NSString *BSGBreadcrumbTypeValue(BSGBreadcrumbType type);
FOUNDATION_EXPORT BSGBreadcrumbType BSGBreadcrumbTypeFromString(NSString * _Nullable value);
```